_datascience/proof.md

```diff
@@ -26,12 +26,23 @@ primary_reviewers: vortexing, abbywall
 ## Release Notes
 See our release notes in the PROOF repo for more details: 
 
-### May 2024
+### June 2024 - PROOF v1.0
 **What's New**
+- Users can now interact with data stored in S3 buckets within WDL workflows executed in PROOF! Just make sure to have your AWS CLI credentials [established in Rhino](https://sciwiki.fredhutch.org/scicomputing/access_credentials/#configure-aws-cli) and you can use the S3 path just like any other input.
+- GPU analysis is now possible via the `gpu` argument in your WDL task's runtime section! For an example, try running [this example script](https://github.com/getwilds/ww-test-workflows/blob/main/gpuMatrixMult/gpuMatrixMult.wdl) in the [ww-test-workflows](https://github.com/getwilds/ww-test-workflows) repo of the [DaSL WILDS](https://github.com/getwilds).
+- Additional features have been introduced to increase transparency in terms of the functionality of PROOF:
+    - New `/info` API endpoint provides details on which code base is being used.
+    - Server start date and time is now available on the "PROOF Server" tab.
+    - Documentation links have been added to the "Welcome" tab and automated server creation email for increased visibility.
 
 **Fixes**
+- The default location of the Apptainer cache directory has been moved from `scratch` to `/hpc/temp` to avoid previously reported linkage issues associated with `scratch`.
```

Comment:
Only for users whose lab/group has a directory in /hpc/temp.

Reply:
Great call again, just added a sub-bullet that describes that, i.e. defaults to home directory if the user's lab/group doesn't have /hpc/temp yet. Thanks Dan!!

Reply:
Actually it still uses scratch if user does not have /hpc/temp yet.